test/e2e/specs/links.test.js

```diff
@@ -432,4 +433,37 @@ describe( 'Links', () => {
 		const assertiveContent = await page.evaluate( () => document.querySelector( '#a11y-speak-assertive' ).textContent );
 		expect( assertiveContent.trim() ).toBe( 'Warning: the link has been inserted but may have errors. Please test it.' );
 	} );
+
+	it( 'link popover remains visible after a mouse drag event', async () => {
+		// Create some blocks so we components with event handlers on the page
+		for ( let loop = 0; loop < 5; loop++ ) {
+			await insertBlock( 'Paragraph' );
+			await page.keyboard.type( 'This is Gutenberg' );
+		}
+
+		// Focus on first paragraph, so the link popover will appear over the subsequent ones
+		await page.mouse.click( 120, 280 );
```

Comment:
This test assumes things are in given position, I think it would have been better to rely on the actual position of things (See `boundingBox`)

Reply:
Or, since it may be complex, alternative means to navigate to the intended block:

- Shift-Tab?
- Arrow Up?
- Block navigator?